bambi/plots/plot_cap.py

```diff
@@ -155,6 +156,8 @@ def plot_cap(
         are the names of the variables.
     target : str
         Which model parameter to plot. Defaults to 'mean'.
```

Comment:
I would add this only has effect when `pps` is `False`

Reply:
I was also thinking about this and agree. I will also add error handling so that if users do attempt to use `pps=True` and `target` is not None, it will raise an informative error.

Reply:
Good! But I don't think it should be an error, because it's not something we cannot recover from. A warning would be more appropriate. However, the default is `target="mean"` so it would cause the warning unless users pass `target=None` explicitly... So finally, I wouldn't make it either an error or a warning. Just the note in the docstring is fine.